CMakeLists.txt

```diff
@@ -3,6 +3,10 @@ cmake_minimum_required(VERSION 3.15) # 3.15 for MSVC_RUNTIME_LIBRARY, removing d
 cmake_policy(SET CMP0091 NEW)
 cmake_policy(SET CMP0092 NEW)
 
+if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.20)
```

Comment:
Is there a reason not to use `if(POLICY CMP0115)`

Reply:
Also i think vcpkg could simply do:
`cmake_minimum_required(VERSION 3.15...3.24)`

Reply:
I think you can then remove all the explicit settings of `cmake_policy`